cadence-style-guide.md

```diff
@@ -0,0 +1,60 @@
+# Cadence Style Guide
+## Table of contents
+1. [Indentation](#indentation)
+1. [Line length](#line-length)
+1. [Comments](#comments)
+1. [Whitespace between code lines](#whitespace-between-code-lines)
+1. [Declaring variables and constants](#whitespace-between-code-lines)
+1. [Bracket spaces](#bracket-spaces)
+1. [Panic messages](#panic-messages)
+1. [Naming and capitalization](#naming-and-capitalization)
+## Indentation
+Use 4 spaces per indentation level.
+Prefer spaces over tabs as indentation method, mixing them should be avoided.
+## Line length
+A line length of 80 characters is recommended for Cadence. This could be particularly challenging when writing capability-related lines, for instance:
+```cadence
+letResourceRef = account.getCapability(ContractName.SomePathName).borrow<&SomeResource{ContractName.InterfaceName}>()
+```
+This generic line of code that can be found in many transactions that need to borrow a reference to a certain object from a capability is 118 characters long by its own without any indentation. Keeping this line readable should be an objective of any Cadence developer. There are some simple patterns that could be observed in order to accomplish this:
```

Comment:
Add line breaks after logical ends, i.e. put logical parts of the paragraph on separate lines:

```suggestion
The following line of code is an example of code that can be found in many transactions.
It borrows a reference to a certain object from a capability, and is 118 characters long without any indentation. 
Keeping this line readable should be an objective of any Cadence developer. 
There are some simple practices that should be followed in order to accomplish this:
```